packages/user-management-widget/src/lib/widget/state/selectors.ts

```diff
@@ -12,9 +13,18 @@ export const getUsersList = createSelector(getRawUsersList, (users) =>
   users.map((user) => ({
     ...user,
     status: userStatusMappings[user.status] || user.status,
+    roles: user.roleNames
+      .concat(user.userTenants?.[0]?.roleNames)
+      .filter(Boolean)
```

Comment:
I think you can concat an empty array if there is no roleNames and eliminate the filter